sympy/stats/joint_rv.py

```diff
@@ -330,3 +351,12 @@ def marginalise_out(self, expr, rv):
 
     def __call__(self, *args):
         return self.pdf(*args)
+
+class StochasticProcess(JointDistribution):
```

Comment:
I don't think this is a joint distribution in the end. I should behave like a joint distribution in some cases.

Reply:
I think it should rather be like collection of joint distributions. It should have a method creating a joint distribution for each finite set of indices.